src/queries/a11y-hint.js

```diff
@@ -0,0 +1,25 @@
+import { queryAllByProp, buildQueries, getPropMatchAsString } from './all-utils';
+
+const queryAllByA11yHint = queryAllByProp.bind(null, 'accessibilityHint');
+
+const getMultipleError = (c, hint) =>
+  `Found multiple elements with the accessibilityHint of: ${hint}`;
+const getMissingError = (c, hint) =>
+  `Unable to find an element with the accessibilityHint of: ${hint}`;
```

Comment:
Let me think about this one. I’ll definitely fix the other two, my only concern with this one is maintainability (with regards to keeping in parity with dom-testing-library). Whenever there’s a big refactor over there, it might make it hard to pull stuff over here. 

Right now, it’s easy to keep up to date with the rest of the family because it’s so similar. I think at some point we’ll have to decide if we wanna split from *-testing-library on code style like this. I don’t think it’s bad so long as the API and feel for users is the same, it just might make it tough to keep up.